mithril-client/src/entities.rs

```diff
@@ -21,10 +21,6 @@ pub struct Config {
 /// for the purpose of tabular display
 #[derive(Table, Debug, Clone, PartialEq, Eq, PartialOrd, Serialize)]
 pub struct SnapshotListItem {
-    /// Cardano network
```

Comment:
We could have kept this information of the Cardano network and use the `snapshot.beacon.network` to retrieve the value